kuma/static/js/wiki-edit.js

```diff
@@ -458,90 +458,120 @@
     // Initialize logic for save and save-and-edit buttons.
     //
     function initSaveAndEditButtons () {
+        var $form = $('#wiki-page-edit');
+        // Handle edits on the translate page
+        if ($form.length == 0 && $('#translate-document').length == 1) {
+            $form = $('#translate-document');
+        }
+
         // Save button submits to top-level
-        $('.btn-save').on('click', function () {
-            if (draftingEnabled) {
+        $form.on('submit', function () {
+            if (supportsLocalStorage) {
                 // Clear any preserved content.
-                clearDraft('publish');
+                clearDraft();
```

Comment:
Please keep ` clearDraft('publish');` 

(The clearDraft() functionality was augmented after the branch you forked this from).